pydynamo_brain/pydynamo_brain/ui/radiiPainter.py

```diff
@@ -83,63 +84,98 @@ def drawPoint(self, point, selectedPointID):
             )
             self.maybeDrawText(x, y, point)
 
-    def returnRadiusCoord(self, point, radius):
+    def returnRadiusCoord(self, point, radius, realRadius):
         #point radius will be drawn from
         delta = 1
-
-        """        #previous point
-        posX, posY, negX, negY = x2, y2, x2, y2,"""
+        #Drawing the radius calipers for root points
         if  point.isRoot():
-            #print(point.isRoot)
             nextPoint  = point.nextPointInBranch(delta)
-            #print(nextPoint)
             p2Loc = point.location
-            #print(p2Loc)
-            x2, y2, z2 = self.zoomedLocation(p2Loc)
 
+            x2, y2, z2 = self.zoomedLocation(p2Loc)
             p1Loc = nextPoint.location
             x1, y1, z1 = self.zoomedLocation(p1Loc)
 
-
-        else:
+            #length between two points
+            L = math.sqrt(math.pow((x2-x1),2)+math.pow((y2-y1),2))
```

Comment:
if you use:
`x, y, _ = util.normDelta((x2, y2, 0), (x1, y1, 0))`

x will be (x1 - x2)/L, y will be (y1 - y2)/L